.github/workflows/build.yml

```diff
@@ -33,6 +35,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip setuptools
           python -m pip install .[dev,publishing]
+          python -m pip install -r requirements.txt
```

Comment:
Once the dependencies have been specified in pyproject.toml, the requirements.txt file and this line should be removed.
```suggestion
```